scrapd/core/apd.py

```diff
@@ -15,6 +18,7 @@
 PAGE_DETAILS_URL = 'http://austintexas.gov/'
 
 
+@retry(stop=stop_after_attempt(3), wait=wait_exponential(multiplier=1, min=4, max=10))
```

Comment:
Have we seen any examples of >3 failures? I'm not sure how rare it is but that seems safe to me.

Reply:
We have not because it currently fails at the first one. If we see more failures, I'll readjust the parameters (more retries, more wait).